### ISSUE TITLE
onMessageReceived was never invoked 

### ISSUE THREAD

**Issue by ShongSu** [evollu/react-native-fcm#599]
react-native: 0.47.2 
react-native-fcm: 9.7.2
Android 6.0.1 Samsung Tablet

I am monitoring `onMessageReceived` function in logcat from android studio.

For notification type payload,
- Foreground: it should call `onMessageReceived` but it's not
- Background: `onMessageReceived` was not invoked but I am able to get notification banner which is generated by system. That is what we expected.
- No running: same action to Background.

For data only payload,
- Foreground: it should call `onMessageReceived` but it's not
- Background:  it should call `onMessageReceived` but it's not
- No running:  it should call `onMessageReceived` but it's not

In summary, it seems the `onMessageReceived` was never invoked in any situation. I have no idea what's wrong with it. Any suggestion?

**Comment by evollu** [comment]
network issue? notification permission? manifest config? restart device?
